template.go

```diff
@@ -0,0 +1,90 @@
+package main
```

Comment:
Can you move this into utils subpackage? I want to start moving some code out of the main file to improve the overall structure so it's probably better to try applying that rule for all new code

Reply:
I thought about the same thing. We can move every job into a separate file first, split job functions into smaller ones for readability and then think about subpackaging.